Assignment-2/src/index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>2nd-Responsive</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+
+  <div class="menu">
+    <main>
+      <h1>CAMPER CAFE</h1>
+      <p class="established">Est. 2023</p>
+
+      <hr>
+
+      <section>
+        <h2>Coffee</h2>
+
+        <img src="coffee.jpg" alt="Coffee">
+        <article class="item">
+          <p class="flavor">French Vanilla</p>
+          <p class="price">3.00</p>
+        </article>
+
+        <article class="item">
+          <p class="flavor">Caramel Macchiato</p>
+          <p class="price">3.75</p>
+        </article>
+
+        <article class="item">
+          <p class="flavor">Pumpkin Spice</p>
+          <p class="price">3.50</p>
+        </article>
+
+        <article class="item">
+          <p class="flavor">Hazelnut</p>
+          <p class="price">4.00</p>
+        </article>
+
+        <article class="item">
+          <p class="flavor">Mocha</p>
+          <p class="price">4.50</p>
+        </article>
+
+      </section>
+
+      <section>
+        <h2>Desserts</h2>
+        <img src="https://cdn.freecodecamp.org/curriculum/css-cafe/pie.jpg" alt="pie icon" />
+
+        <article class="item">
```

Comment:
Question: Do you know what we call these tags? `section`, `article` and others like this?